### DISCUSSION TITLE
Support for High DPI screens ?

### DISCUSSION THREAD

**Discussion by Nickmcgit** [mRemoteNG/mRemoteNG#2573] [Ideas]
Will there be support for high DPI screens in the future. I have a multi screen setup where two screens are at 100% window scaling and my laptop screen which is higher density is at 125%. mRemoteNG text looks quite blurry and ugly on this screen because it's not window scaling properly. Will there be support for this in the future ?

Here's an example image of the problem. We see that powershell scales proerly to the 1920x1080 screen at 125% window scaling but mRemoteNG doesn't.

![mremoteng_text](https://github.com/mRemoteNG/mRemoteNG/assets/144028093/fdf880f6-df01-4adf-bcfc-808414363b9b)

**Comment by Kvarkas** [comment]
issue is with limitations of .Net 6 framework related to WinForms, after release I am planning to start migration to .Net 8 where this issue seems to be properly addressed already

**Reply by Nickmcgit** [reply]
Good to know Kvarkas...
